data/json/monstergroups.json

```diff
@@ -1359,8 +1359,8 @@
       { "monster" : "mon_zombie_master", "freq" : 10, "cost_multiplier" : 15 },
       { "monster" : "mon_beekeeper", "freq" : 5, "cost_multiplier" : 1 },
       { "monster" : "mon_zombie_technician", "freq" : 10, "cost_multiplier" : 6},
-      { "monster" : "mon_zombie_brute_shocker", "freq" : 10, "cost_multiplier" : 20 }
-
+      { "monster" : "mon_zombie_brute_shocker", "freq" : 10, "cost_multiplier" : 10 },
```

Comment:
**!**
*KA101 hurriedly makes a radio call*
I've spotted a stealth rebalance!  Send reinforcements!

(To be clear: nope, changing spawn levels that specifically should at least be specifically mentioned.  Ideally they'd be their own PR.  I'm not sure it's warranted as a result of adding these folks, so gonna ask you to revert it.)